src/components/Footer.tsx

```diff
@@ -0,0 +1,79 @@
+import classNames from 'classnames';
+import { useMemo } from 'react';
+
+import { Todo } from '../types/Todo';
+import { FilterStatus } from '../types/FilterStatus';
+
+type Props = {
+  todos: Todo[];
+  filterStatus: FilterStatus;
+  onSetFilterStatus: (status: FilterStatus) => void;
+  onClearCompleted: () => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  filterStatus,
+  onSetFilterStatus,
+  onClearCompleted,
+}) => {
+  const activeTodosCount = useMemo(() => {
+    return todos.filter(todo => !todo.completed).length;
+  }, [todos]);
+
+  const noCompletedTodos = useMemo(() => {
+    return todos.every(todo => !todo.completed);
+  }, [todos]);
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${activeTodosCount} items left`}
```

Comment:
Add condition for right text with 1 item left
```suggestion
        {`${activeTodosCount} item${activeTodosCount > 1 ? 's' :  ''} left`}
```